examples/addsvc/main.go

```diff
@@ -78,20 +84,22 @@ func main() {
 		))
 	}
 
-	// package tracing
-	var collector zipkin.Collector
+	// Set up OpenTracing
+	var tracer opentracing.Tracer
 	{
-		zipkinLogger := log.NewContext(logger).With("component", "zipkin")
```

Comment:
I didn't thread a logger through to record errors... let me know if that's desired.